src/lib/sql/sql_to_ast_translator.cpp

```diff
@@ -844,6 +860,48 @@ std::shared_ptr<AbstractASTNode> SQLToASTTranslator::_translate_show(const hsql:
   return {};
 }
 
+std::shared_ptr<AbstractASTNode> SQLToASTTranslator::_translate_create(const hsql::CreateStatement& create_statement) {
+  switch (create_statement.type) {
+    case hsql::CreateType::kCreateView: {
+      auto view = _translate_select((const hsql::SelectStatement&)*create_statement.select);
+      if (create_statement.viewColumns) {
```

Comment:
Please doc why/when (what does the existence of `create_statement.viewColumns` mean?) another ProjectionNode needs to be added at the root.